component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceServiceImpl.java

```diff
@@ -1475,6 +1509,29 @@ private void removeWebNotifications(String spaceId, String userId) {
       LOG.error("Cannot update web notfication. WebNotificationService is null");
     }
   }
+  
+  /**
+   * {@inheritDoc}
+   */
+  public void setRedactor(Space space, String userId, boolean isRedactor) {
```

Comment:
IMO we need to split this and add functions addRedactor, removeRedactor instead of having one single one for both.

Reply:
I have done same as setManager